index.html

```diff
@@ -13,8 +13,24 @@
 
 <body>
     <div class="container">
+   <header>
+   <img src="img/lambda-black.png" alt="logo">
```

Comment:
make sure to keep your opening and closing tag indentations the same. It will save you when it comes to debugging.